content/guides/mobile/ios/index.md

```diff
@@ -0,0 +1,6 @@
+---
+related_pages:
+ - sdks-and-tools
+---
+
+# iOS
```

Comment:
Definitely needs more content. Ideas:

* Image to show what the SDK allows a dev to do?
* Maybe an overview of the SDK enables (and doesn't enable?)

Reply:
Added image + additional text